modules/order/commerce_order.post_update.php

```diff
@@ -53,3 +53,47 @@ function commerce_order_post_update_2() {
     }
   }
 }
+
+/**
+ * Add order `receipt` view mode for `placed` notification emails.
+ */
+function commerce_order_post_update_3() {
+
```

Comment:
Unneeded newline at the beginning of the function. Docblock can be just "Adds the order 'receipt' view mode."